packages/manager/modules/vps/src/secondary-dns/delete/delete-secondary-dns.routing.js

```diff
@@ -1,11 +1,17 @@
 export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('vps.detail.secondary-dns.delete', {
     url: '/delete',
```

Comment:
No need for
```suggestion
    url: '/delete?domain',
```
?

Reply:
I suppose `domain` isn't a string

Reply:
Its an `object`

Reply:
@FredericEspiau @varun257 I'm having trouble understanding the answer here 😄 
What I meant is you don't need to declare that your url is taking a queryParameter ?

Reply:
No, since i add resolve parameter, its not required to add domain as `queryParams`